### PR TITLE
Prevent creating v1 cards

### PR DESCRIPTION
**Pull request by dhaselhan**
* Remove App and NOI creation from all boards
* Hide in Admin UI for Create

### CODE REVIEW

File: alcs-frontend/src/app/features/admin/board-management/board-management-dialog/board-management-dialog.component.ts [hunk 1]
```diff
@@ -98,7 +100,7 @@ export class BoardManagementDialogComponent implements OnInit {
     this.step++;
     //We don't support creating Notifications
     this.allowedCreateCardTypes = (this.permittedCardTypes.value ?? []).filter(
-      (cardType) => cardType !== CardType.NOTIFICATION
+      (cardType) => !DISABLED_CREATE_CARD_TYPES.includes(cardType),
```
Comment (trslater) [inline alcs-frontend/src/app/features/admin/board-management/board-management-dialog/board-management-dialog.component.ts:103]: Won't this now enable `NOTIFICATION` card type? Is that intended?

File: alcs-frontend/src/app/features/admin/board-management/board-management-dialog/board-management-dialog.component.ts [hunk 2]
```diff
@@ -98,7 +100,7 @@ export class BoardManagementDialogComponent implements OnInit {
     this.step++;
     //We don't support creating Notifications
```
Comment (sandratoh) [inline alcs-frontend/src/app/features/admin/board-management/board-management-dialog/board-management-dialog.component.ts:101]: We should probably update this comment too

File: alcs-frontend/src/app/features/admin/board-management/board-management-dialog/board-management-dialog.component.ts (hunk 1 above)
Comment (dhaselhan) [reply alcs-frontend/src/app/features/admin/board-management/board-management-dialog/board-management-dialog.component.ts:102]: It definitely would and not intended. Good catch! MR updated.

File: alcs-frontend/src/app/features/admin/board-management/board-management-dialog/board-management-dialog.component.ts (hunk 2 above)
Comment (dhaselhan) [reply alcs-frontend/src/app/features/admin/board-management/board-management-dialog/board-management-dialog.component.ts]: Comment removed.

**Review by trslater — APPROVED** [review]
LGTM 👍

**Pull request merged**